docs/source/reference/hi.rst

```diff
@@ -0,0 +1,29 @@
+.. _hi:
+
+IMAP-Hi
+==========================
+
+This is the IMAP-Hi Instrument module, which contains the code for processing
+data from the IMAP-Hi instrument.
+
+The L0 code to decommutate the CCSDS packet data can be found below.
+
+.. currentmodule:: imap_processing
+
+.. autosummary::
+    :toctree: generated/
+    :template: autosummary.rst
+    :recursive:
+
+    decom
+
+IMAP-Hi test and validation code can be found below.
```

Comment:
I personally wouldn't worry about documenting the test and validation code, probably especially with the autosummary just because we don't generally put great docstrings into the tests. Maybe just a link to specific test or validation methods that would be useful to point others to (if any)?

Reply:
I'd suggest removing them for now then. I actually think a description of the instrument and what it does is just fine for this start of documentation without any need to reference code here if there is nothing to reference. Just making the placeholder is a win :)